XC Profile/profileJS.js

```diff
@@ -12,20 +26,153 @@ aboutMe.appendChild(pAboutMe);
 let pAboutMeText = document.createTextNode("Hi all, my name is Xochitl and thank you for visting my page and this is my page.");
 pAboutMe.appendChild(pAboutMeText);
 
+//*************ONLINE RESUME*********
+//Add Collapsible Button onlineResume
+let btn2 = document.createElement("BUTTON");
+btn2.setAttribute('class', 'collapsible');
+btn2.setAttribute('id', 'btnonlineResume');
+let btn2Text = document.createTextNode("Online Resume");
+btn2.appendChild(btn2Text);
+document.body.appendChild(btn2);
+
+//Add new Div: onlineResume
+let newDiv2 = document.createElement('div');
+newDiv2.setAttribute('id', 'onlineResume');
+newDiv2.setAttribute('class', 'content')
+document.body.appendChild(newDiv2);
+
 //Adding P to DIV: onlineResume
 let pResume = document.createElement('p');
 pResume.setAttribute('id', 'presume');
 onlineResume.appendChild(pResume);
 let pResumeText = document.createTextNode("Work experience: Short description");
 pResume.appendChild(pResumeText);
 
+//*************CODE EXPERIENCE*********
+//Add Collapsible Button Code Experience
+let btn3 = document.createElement("BUTTON");
```

Comment:
These variable names could be named something a bit more descriptive. I like to use variable names as documentation.

For example, instead of `btn3` you could write `codeExperienceButton`